packages/vector_graphics/lib/src/listener.dart

```diff
@@ -225,15 +228,16 @@ class FlutterVectorGraphicsListener extends VectorGraphicsCodecListener {
     final List<ui.Color> colorValues = <ui.Color>[
       for (int i = 0; i < colors.length; i++) ui.Color(colors[i])
     ];
+    final bool hasFocal = focal != center && focal != null;
     final ui.Gradient gradient = ui.Gradient.radial(
       center,
       radius,
       colorValues,
       offsets,
       ui.TileMode.values[tileMode],
-      null,
-      focal,
-      radius,
+      transform,
+      hasFocal ? focal : null,
+      hasFocal ? radius : 0,
```

Comment:
This should unconditionally be zero.

SVG does not support focal radius. Flutter/Skia does. If we want our format to support it then we need another parameter.

Reply:
done